addons/web_tour/static/src/tour_service/tour_interactive.js

```diff
@@ -60,6 +61,13 @@ export class TourInteractive {
         }
     }
 
+    findTriggers(trigger) {
+        const parts = trigger.split(/,\s*(?![^(]*\))/);
```

Comment:
Just a thought here.
comma `,` has a signification in CSS, different from the one here.
How about using explicit double-pipes `||` ?
And crash if the expression is too complex and contains parenthesis and other things (in an attempt of the developper to have complex conditions) ? (in debug mode ?)